pkg/plugins/platform/local.go

```diff
@@ -47,6 +47,7 @@ func (l *LocalApp) GetType() string {
 func (l *LocalApp) Init(basePath string) error {
 	config, err := ddevapp.NewConfig(basePath, "")
 	if err != nil {
+		l.AppConfig = config
```

Comment:
Add a comment above this saying why we're saving it away, so somebody doesn't remove this someday.